requirements/static/ci/common.in

```diff
@@ -49,3 +49,7 @@ genshi>=0.7.3
 cheetah3>=3.2.2
 mako
 wempy
+impacket
+pyspnego==0.9.2; sys_platform != 'win32' and python_version <= '3.7'
+pyspnego==0.10.2; sys_platform != 'win32' and python_version >= '3.8'
+smbprotocol==1.10.1; sys_platform != 'win32'
```

Comment:
```suggestion
pyspnego<0.10.0; sys_platform != 'win32' and python_version <= '3.7'
pyspnego>=0.10.0; sys_platform != 'win32' and python_version >= '3.8'
smbprotocol>=1.10.1; sys_platform != 'win32'
```
Why are these locked in the `.in` file?

Reply:
Also, why don't we want these extensions installed on windows?

Reply:
samba is non-windows way to access Windoes, so not needed on Windows since it accesses it's file system natively, first had to deal with samba when working on NetWare, that's why I don't just state  Linux.

Also pyspnego post version 0.9.2 they dropped support for Python 3.7 since it is EOL, hence the need for the Python 3.7 check

Note these were already in the cloud.in file, now I need them for smb testing, moved to common.in and adjusted since don't want these on Windows.

Rejecting suggestion

Reply:
Must have missed the impacket change I made constraining it, since it fails to load on Windows.  That was in the PR building too
pip-tools - don't think so.

Reply:
doing it